yarn.lock

```diff
@@ -1389,7 +1389,6 @@
 
 "@matrix-org/olm@https://gitlab.matrix.org/api/v4/projects/27/packages/npm/@matrix-org/olm/-/@matrix-org/olm-3.2.14.tgz":
   version "3.2.14"
-  uid acd96c00a881d0f462e1f97a56c73742c8dbc984
```

Comment:
this looks like an unrelated change

Reply:
You're right, I'm not sure why yarn (1.22.19, latest as of now) decided to do that. I only `yarn remove`d qs and committed whatever it spat out.

I'll revert the line, but it may crop back up later, I genuinely have no idea. `@matrix-org/olm` seems to be the only entry in that file that has a `uid` value.